src/edu/stanford/rsl/tutorial/weightedtv/Perform2DwTV.java

```diff
@@ -0,0 +1,293 @@
+package edu.stanford.rsl.tutorial.weightedtv;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import ij.IJ;
+import ij.ImageJ;
+import ij.ImagePlus;
+import edu.stanford.rsl.tutorial.filters.RamLakKernel;
+import edu.stanford.rsl.conrad.data.numeric.Grid1D;
+import edu.stanford.rsl.conrad.data.numeric.Grid2D;
+import edu.stanford.rsl.conrad.data.numeric.NumericPointwiseOperators;
+import edu.stanford.rsl.conrad.geometry.Rotations;
+import edu.stanford.rsl.conrad.geometry.transforms.ScaleRotate;
+import edu.stanford.rsl.conrad.numerics.SimpleMatrix;
+import edu.stanford.rsl.conrad.utils.ImageUtil;
+import edu.stanford.rsl.tutorial.phantoms.Phantom;
+import edu.stanford.rsl.tutorial.phantoms.SheppLogan;
+import edu.stanford.rsl.tutorial.fan.FanBeamProjector2D;
+import edu.stanford.rsl.tutorial.fan.FanBeamBackprojector2D;
+
+//import TVreconstruction.FanBeamProjector2D;
+//import edu.stanford.rsl.tutorial.fan.FanBeamProjector2D;
+
+/**
+ *[1]Candes. Enhancing Sparsity by Reweighted l1 Minimization
+ * 
+ * 
+ * @author Yixing Huang
+ *
+ */
+public class Perform2DwTV {
+	static boolean isOpencl=true;
+	
+	
+	 private static String path="D:\\wTV\\";
```

Comment:
This will Create a log file at a hard coded location. If you desperately want to use a log file, you should create a registry key and generate the log file there. In fact, we could do this in a function called Conrad.log() and also specify debug levels. This would be a good extension to the framework...